packages/office-ui-fabric-react/src/utilities/keytips/KeytipManager.ts

```diff
@@ -0,0 +1,430 @@
+import { KeytipLayerBase } from '../../KeytipLayer';
+import { KeytipTree } from './KeytipTree';
+import { IKeytipTreeNode } from './IKeytipTreeNode';
+import { IKeytipProps } from '../../Keytip';
+import {
+  IKeySequence,
+  convertSequencesToKeytipID,
+  transitionKeysContain,
+  IKeytipTransitionKey,
+  fullKeySequencesAreEqual,
+  getDocument,
+  replaceElement,
+  findIndex,
+  mergeOverflowKeySequences,
+  Async
+} from '../../Utilities';
+import { constructKeytipExecuteTargetFromId } from './KeytipUtils';
+
+export interface IUniqueKeytip {
+  uniqueID: string;
+  keytip: IKeytipProps;
+}
+
+export class KeytipManager {
```

Comment:
[nit] might be nice to add a class comment  with  short description of what this class does